lib/iris/_constraints.py

```diff
@@ -73,6 +73,26 @@ def __init__(self, name=None, cube_func=None, coord_values=None, **kwargs):
                        model_level_number=[10, 12])
                        & Constraint(ensemble_member=2)
 
+        .. note::
+            Whilst ``&`` is supported, the ``|`` that might reasonably be expected
+            is not. This is because each constraint describes a boxlike region, and
+            thus the intersection of these constraints (obtained with ``&``) will
+            also describe a boxlike region. Allowing the union of two constraints
+            (with the ``|`` symbol) would allow the description of a non-boxlike
+            region. These are difficult to describe with cubes and so it would be
+            ambiguous what should be extracted.
+
+            To generate multiple cubes, each constrained to a different range of
+            the same coordinate, use :py:func:`iris.load_cubes`.
```

Comment:
```suggestion
            the same coordinate, use :py:func:`iris.load_cubes` or :meth:`iris.cube.CubeList.merge_cubes`.
```

Showing that constraints are used in the same way for extracting as for loading.

Reply:
I can't see a method called "iris.cube.CubeList.merge_cubes" in the docs, am I missing something? Did you mean "extract_cubes"?

Reply:
Sorry, yes, I did mean 'extract_cubes' .
And I was _so_ careful to check the import path, too ...

Reply:
Committed this addition